new/postinstall.sh

```diff
@@ -17,6 +17,12 @@ php $PATCHDEMO/wikis/$NAME/w/maintenance/createAndPromote.php Mallory patchdemo1
 # This command may fail as --disable-autoblock was only added in 1.36, so suppress errors
 echo "Mallory" | php $PATCHDEMO/wikis/$NAME/w/maintenance/blockUsers.php --reason "Blocking account for testing" --disable-autoblock || echo "Can't block Mallory"
 
+# run arbitrary SQL
+for sql in $(find $PATCHDEMO/sql-perwiki -name "*.sql" -not -type d -printf '%P\n')
+do
+	mysql -u patchdemo -ppatchdemo patchdemo < $PATCHDEMO/sql-perwiki/$sql
```

Comment:
The database name for the wiki is patchdemo_$NAME

Reply:
Fixed